src/CLI/clitree/cli-xml/bgp.xml

```diff
@@ -451,8 +456,13 @@ limitations under the License.
     <ACTION> python $SONIC_CLI_ROOT/sonic-cli-bgp.py patch_openconfig_network_instance_network_instances_network_instance_protocols_protocol_bgp_neighbors_neighbor_config_description ${vrf-name} ${nbr-addr} ${String}</ACTION>
   </COMMAND>
   <COMMAND name="ebgp-multihop" help="Allow EBGP neighbors not on directly connected networks">
-    <PARAM name="hop-count" help="Maximum hop count (default=1)" ptype="RANGE_1_255"/>
-    <ACTION> python $SONIC_CLI_ROOT/sonic-cli-bgp.py patch_openconfig_network_instance_network_instances_network_instance_protocols_protocol_bgp_neighbors_neighbor_ebgp_multihop_config ${vrf-name} ${nbr-addr} True ${hop-count}</ACTION>
+    <PARAM name="hop-count" help="Maximum hop count (default=255)" ptype="RANGE_1_255" optional="true"/>
+    <ACTION> if test -n "${hop-count}" ; then&#xA; 
```

Comment:
Add default=255 attribute to PARAM and then ${hop-count} will always have a value. Then this test can be avoided.